modules/ROOT/pages/vm/vm-publish-listen.adoc

```diff
@@ -1,25 +1,27 @@
-= Example: To Publish and Listen to Messages through the VM Connector - Mule 4
+= Example: Publish and Listen to Messages through the VM Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-This example shows how to publish a message that is received by a different flow.
+The following example shows how to publish a message that is received by a different flow. To do this you use the `<vm:publish>` and `<vm:listener>` elements that point to the same queue.
```

Comment:
Add a comma after "this." I suggest removing "the": 

"To do this, use `<vm:publish>` and `<vm:listener>` elements that point to the same queue."